cmd/dra-example-kubeletplugin/main.go

```diff
@@ -33,9 +33,7 @@ import (
 )
 
 const (
-	PluginRegistrationPath     = "/var/lib/kubelet/plugins_registry/" + consts.DriverName + ".sock"
 	DriverPluginPath           = "/var/lib/kubelet/plugins/" + consts.DriverName
```

Comment:
It's good practice to have a configuration option for `/var/lib/kubelet/plugins` and `/var/lib/kubelet/plugins_registry`. That makes a DRA driver usable on Kubernetes distros with unusual paths.

This wasn't done before, so this can also be enhanced in a follow-up PR.